auditbeat/module/auditd/_meta/fields.yml

```diff
@@ -858,7 +858,8 @@
     type: group
     description: >
       The geoip fields are defined as a convenience in case you decide to
-      enrich the data using a geoip filter in Logstash or Ingest Node.
+      enrich the data using a geoip filter in Logstash or Elasticsearch geoip
```

Comment:
I don't think these yaml files deal very well with strange characters (like backticks and curly braces) so it's best to omit them.